pkg/kube/client.go

```diff
@@ -77,10 +77,12 @@ func New(config clientcmd.ClientConfig) *Client {
 	return &Client{
 		Factory:        cmdutil.NewFactory(config),
 		SchemaCacheDir: clientcmd.RecommendedSchemaFile,
-		Log:            func(_ string, _ ...interface{}) {},
+		Log:            nopLogger,
 	}
 }
 
+var nopLogger = func(_ string, _ ...interface{}) {}
```

Comment:
Out of curiosity, is this just cleanup or something that's necessary? Love it

Reply:
it is now reused in client_test.go